### ISSUE TITLE
Add a `FsSql.Npgsql` package that registers `NpgSqlCommand` as supporting async ops

### ISSUE THREAD

**Issue by baronfel** [irium/FsSql#4]
Would need to be a new package so that we don't have `Npgsql` as a dependency of the main project.

**Comment by irium** [comment]
Maybe because of I'm not native english, but I don't understand what you mean. Currently there is no any dependency on `Npgsql` at all. If you mean to create new project and package - feel free to do it. I can mention it in the `README` if you want.

**Comment by baronfel** [comment]
Yep, that's all I mean. I ran into this today and mostly wanted to create an issue in case someone else had seen this project too!  We're working on a package at work to do what I mentioned above and wehn we get something more solid I'll write back.
